tfjs-core/src/util_test.ts

```diff
@@ -665,3 +665,58 @@ describe('util.decodeString', () => {
     expect(util.isPromise(promise3)).toBeFalsy();
   });
 });
+
+describe('setTimeout', () => {
+  // If we set a larger number here, an error will be reported as "'expect'
+  // was used when there was no current spec, this could be because an
+  // asynchronous test timed out".
```

Comment:
This is because Jasmine does not know you're still testing `setTimeout`, so the `expect` called in the callback is actually called during another test (or after all tests are complete, which causes the error you're seeing). Marking the test as `async` or using a `done` callback should fix this. [Docs](https://jasmine.github.io/tutorials/async)

Reply:
Thanks for the suggestion. I used done callback to solve this.